internal/elasticsearch/client.go

```diff
@@ -86,40 +78,34 @@ type Client interface {
 	ListTemplates() (sets.String, error)
 	GetIndexTemplates() (map[string]estypes.GetIndexTemplate, error)
 	UpdateTemplatePrimaryShards(shardCount int32) error
-
-	SetSendRequestFn(fn FnEsSendRequest)
 }
 
-type FnEsSendRequest func(cluster, namespace string, payload *EsRequest, client k8sclient.Client)
-
 type esClient struct {
-	cluster         string
-	namespace       string
-	k8sClient       k8sclient.Client
-	fnSendEsRequest FnEsSendRequest
+	cluster   string
+	namespace string
+	eoclient  elasticsearch6.Client
```

Comment:
can we call this something else, maybe just `client` ?

Reply:
yes we can .. I will change and push latest changes.